### PR TITLE
update caption

### PR DESCRIPTION
**Pull request by NoorChasib**

### CODE REVIEW

File: _slides/NoorChasib.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+```
+
```
Comment (martins-vds) [inline _slides/NoorChasib.md:2]: comment

File: _slides/NoorChasib.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Meow meow meow
+
+
+
+
+crlf
```
Comment (martins-vds) [inline _slides/NoorChasib.md:8]: another comment

File: _slides/NoorChasib.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+
+
+
```
Comment (martins-vds) [inline _slides/NoorChasib.md:12]: yet another comment

File: _slides/NoorChasib.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+```
+
+Meow meow meow
```
Comment (juhewitt) [inline _slides/NoorChasib.md:3]:
``` suggestion
Meow meow meow goes the cat

File: _slides/NoorChasib.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+
+
+
+crlf
+
+
```
Comment (martins-vds) [inline _slides/NoorChasib.md:10]:
```suggestion
Change me
```

**Review by martins-vds — APPROVED** [review]

**Pull request closed without merging**